### PR TITLE
feat: Cart Summary & Category in home page

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
1. added cart summary
2. fixed bugs in wishlist (in wishlist icon)
3. Category feature in home page

### CODE REVIEW

File: src/pages/Cart/index.js [hunk 1]
```diff
@@ -10,9 +10,12 @@ import {HorizontalCard} from "../../components/common/HorizontalCard";
 import { PriceDetails } from "../../components/Cart/priceDetails";
 
 export const Cart = () => {
-  const { state, dispatch } = useWishList();
+  const { state, dispatch, cartSummary } = useWishList();
 
   useEffect(() => getCartItems(dispatch), []);
+
+  const cartQuantity = cartSummary(state.cartData)
+    console.log(cartQuantity)
```
Comment (prathmeshjagtap) [inline src/pages/Cart/index.js]: remove console.log

**Review by prathmeshjagtap — APPROVED** [review]

File: src/components/Cart/priceDetails.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
 import { useWishList } from "../../context/wishlistContext";
 
 export const PriceDetails = () => {
-    const {state} = useWishList()
-    const cartQuantity = state.cartData.length + state.cartQuantity
+    const {state, cartSummary} = useWishList()
+    const cartDetails = cartSummary(state.cartData)
    
   return (
     <div class="price-details-container flex-column">
```
Comment (ArjunGTX) [inline src/components/Cart/priceDetails.js]: replace all occurances of `class` with `className`

File: src/data/categoryData.js [hunk 3]
```diff
@@ -9,13 +9,13 @@ const categoryData = [
         img_name: "Jewellery",
         img_url: "https://res.cloudinary.com/debanftke/image/upload/v1647432178/j3_b59awd.png"
     },
+    // {
```
Comment (ArjunGTX) [inline src/data/categoryData.js:12]: Why is this piece of code commented?

Comment (Vishruta-Patil) [reply src/data/categoryData.js:12]: For now, I have kept 3 categories and will implement more categories in future

**Pull request merged**